NetKAN/ISRUThanksButNoTanks.netkan

```diff
@@ -7,11 +7,14 @@ x_netkan_epoch: 1
 license: MIT
 resources:
   homepage: https://forum.kerbalspaceprogram.com/index.php?/topic/204750-*
+tags:
+  - config
+  - resources
 depends:
   - name: ModuleManager
+recommends:
+  - name: KerbalChangelog
+  - name: StockalikeMiningExtension
```

Comment:
I believe this should be supports

Reply:
The forum thread mentions them as recommends and supports:
![grafik](https://user-images.githubusercontent.com/28812678/132693404-6d905f30-9eb7-4732-ab7c-4f279b7cdb9b.png)

I guess we can put them in as both if you want. `supports` will make ISRUThanksButNoThanks show up when someone installs KerbalChangelog or StockalikeMiningExtension, `recommends` will make KerbalChangelog or StockalikeMiningExtension show up when someone installs ISRUThanksButNoThanks.